### PR TITLE
[MenuBar] Add Cypress and Chromatic tests

### PR DESCRIPTION
**Pull request by andy-hook**
Add Cypress and Chromatic tests

### CODE REVIEW

**Comment by andy-hook** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1846** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1846" target="_blank">Graphite</a> 
    * **PR #1847** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1847" target="_blank">Graphite</a> 
      * **PR #1852** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1852" target="_blank">Graphite</a> 
        * **PR #1881** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1881" target="_blank">Graphite</a> 
          * **PR #1883** <a href="https://app.graphite.dev/github/pr/radix-ui/primitives/1883" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/radix-ui/primitives/1883?utm_source=stack-comment).

File: cypress/integration/Menubar.spec.ts [hunk 1]
```diff
@@ -3,6 +3,267 @@ describe('Menubar', () => {
...
+      pointerOver('Edit');
+      cy.findByText('New Tab').should('not.exist');
+      cy.findByText('Redo').should('not.have.focus');
+
+      pointerOver('History');
+      cy.findByText('Redo').should('not.exist');
+      cy.findByText('Radix').should('not.have.focus');
+    });
+
+    it('should maintain the open state when pointer moves away a trigger', () => {
```
Comment (benoitgrelard) [inline cypress/integration/Menubar.spec.ts:23]:
```suggestion
    it('should maintain the open state when pointer moves away from a trigger', () => {
```

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "test:ci": "yarn types:check && jest --ci && yarn cypress:ci",
     "storybook": "start-storybook -p 9009 --ci",
     "cypress:ci": "start-server-and-test storybook http://localhost:9009 cypress:run",
-    "cypress:run": "cypress run",
+    "cypress:run": "cypress run --spec cypress/integration/Menubar.spec.ts",
```
Comment (benoitgrelard) [inline package.json:17]: leftover?

**Review by benoitgrelard — COMMENTED** [review]
Looks good apart from these couple things! 💯

File: package.json (hunk 2 above)
Comment (andy-hook) [reply package.json]: Yes! thank you

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**